src/App.java

```diff
@@ -12,20 +12,209 @@ public void start() {
         String userName = getUserName();
         printGreeting(userName);
         printInstructions();
-        doMadlib();
-        System.out.println("hi");
+        String choice = choiceMadLibs();
```

Comment:
I like this code - I almost think it would have been better to abstrac this even further by having a method called `chooseAndDoMadlib()` or something that included this if statement, but it works and is cool